documentation/docs/examples/image360.mdx

```diff
@@ -69,14 +69,55 @@ The `transform` property from `Image360` represents the world transformation (ro
 The `Image360Visualization` object has convenience functionality related to the visualization of the 360 image such as setting the opacity of the given 360 image.
   
 ## Events
-The `Image360Collection` exposes events `image360Entered` and `image360Exited` for subscribing to enter & exit from 360 image mode.
+The `Image360Collection` exposes events `image360Entered` and `image360Exited` for subscribing to enter & exit from 360 image mode. `image360Entered` is also fired when the revision of an image is changed.
```

Comment:
Maybe refer to the next section here. As this is the first place in the doc that revisions of 360 images are mentioned. So a first time reader would probably wtf. here.